modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -5,20 +5,26 @@
 [id="installation-osp-kuryr-known-limitations_{context}"]
 = Known limitations of installing with Kuryr
 
-There are known limitations when using Kuryr SDN:
+There are known limitations when using Kuryr SDN, depending on the
+{rh-openstack} version:
 
 * An Amphora load balancer VM is deployed per OpenShift Service with
 the default Octavia load balancer driver (Amphora driver). If the environment is
-resource constrained, creating a large amount of Services could be a problem.
+resource constrained, creating a large amount of Services could be a problem as
+one VM is created for each Service.
```

Comment:
Rewrite:

"A VM is created for each Service. In a resource-constrained environment, creating too many Services can cause problems."

If we want to be specific about the problems that can happen, even if it seems obvious, that's a good move, too.

Reply:
first part of the sentence is repeated in the sentence before that. I'll add the second part